lib/marked.js

```diff
@@ -365,6 +372,10 @@ Lexer.prototype.token = function(src, top) {
           if (!loose) loose = next;
         }
 
+        if (loose) {
+          listStart.loose = true;
```

Comment:
Can this `if` statement be removed in favor of `listStart.loose = loose`

Reply:
no because if `loose` is false  and `listStart.loose` is true I don't want it to change `listStart.loose` to false.

`listStart.loose` is true if **any** of `listStart` items are loose